### DISCUSSION TITLE
Can I use an SSL certificate from a trusted CA on mitmproxy?

### DISCUSSION THREAD

**Discussion by fairchilddb** [mitmproxy/mitmproxy#7113] [Q&A]
Copy question from StackOverflow
https://stackoverflow.com/questions/64256031/can-i-use-an-ssl-certificate-from-a-trusted-ca-on-mitmproxy
When I use a domain name and ssl cert from a trusted ca on mitmporxy, is it possible I can interpret ssl traffic without getting ssl cert not from trust ca like using mitmproxy generated ca？

**Comment by mhils** [comment]
I think this is already answered in my answer on StackOverflow? https://stackoverflow.com/a/64261052/934719

TL;DR: Yes.
